samples/model-builder/deploy_model_custom_trained_model_sample.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from google.cloud import aiplatform
+from typing import Optional, Sequence, Tuple, Dict
+from google.cloud.aiplatform import explain
+
+#  [START aiplatform_sdk_deploy_model_custom_trained_model_sample]
+def deploy_model_custom_trained_model_sample(
+    project,
+    location,
+    model_name: str,
+    machine_type: str,
+    endpoint: Optional["Endpoint"] = None,
+    deployed_model_display_name: Optional[str] = None,
+    traffic_percentage: Optional[int] = 0,
+    traffic_split: Optional[Dict[str, int]] = None,    
+    min_replica_count: Optional[int] = 1,
```

Comment:
Please remove the Optional type hint (just leave `int`) on `min_replica_count` and `max_replica_count`. After reviewing this, I realized Endpoint.deploy and Model.deploy have inconsistent types on those arguments. I opened a PR to fix the function type hints in the SDK here: https://github.com/googleapis/python-aiplatform/pull/345

Reply:
Done.